app/lib/project/MetapolatorProject.js

```diff
@@ -588,11 +588,10 @@ define([
                                this.fontinfoFileName, this.fontinfoFile );
     };
 
-    function exportInstance(io, project, masterName, instanceName, precision) {
+    function exportInstance(io, project, instanceName, dirName, precision) {
```

Comment:
Here is a conceptual problem. Within the metapolaor engine code (i.e. everything in master) there are no instances that you can `project.open(instanceName)` There are only masters, nothing else. However, you can export an instance of an master, meaning that you write it's current state to a file.

The UI (i.e. everything in gh-pages) makes up different meanings of masters and instances. I think this is to please the users and how they see the things. What the UI calls an instance is a master in the engine. It's in fact a MOM-Master the only difference is how it's CPS is being set up.

Please keep `masterName` as argument name.